### ISSUE TITLE
Why set person mask = 0 ?

### ISSUE THREAD

**Issue by Shuai-Xie** [Daniil-Osokin/lightweight-human-pose-estimation.pytorch#44]
When preprocessing the COCO data,

```py
mask = np.ones(shape=(label['img_height'], label['img_width']), dtype=np.float32)
mask = get_mask(label['segmentations'], mask)

def get_mask(segmentations, mask):
    for segmentation in segmentations:  # RLE, multi person
        rle = pycocotools.mask.frPyObjects(segmentation, mask.shape[0], mask.shape[1])
        mask[pycocotools.mask.decode(rle) > 0.5] = 0
    return mask
```
` mask[pycocotools.mask.decode(rle) > 0.5] = 0` will make person masks equal to 0, while bg pixels are still 1.

**I mean, why reverse 0 and 1 ?**

**And in loss function, this mask will just choose the background ???**

```py
def l2_loss(input, target, mask, batch_size):
    loss = (input - target) * mask
    loss = (loss * loss) / 2 / batch_size
    return loss.sum()
```

**Comment by Daniil-Osokin** [comment]
If so, why the model converges? :) This code masks 'crowd' regions. In these regions keypoints aren't properly annotated, so just skip them.

**Comment by Daniil-Osokin** [comment]
Hope, this helped.
